src/main/java/edu/cornell/kfs/concur/ConcurKeyConstants.java

```diff
@@ -35,7 +35,7 @@ public class ConcurKeyConstants {
 
     public static final String CONCUR_REQUEST_EXTRACT_CASH_ADVANCE_INVALID_UNIQUE_IDENTIFIER = "validation.error.concur.request.extract.cash.advance.invalid.unique.identifier";
     public static final String CONCUR_SAE_NOT_REQUESTED_CASH_ADVANCE_DATA_LINE = "validation.error.concur.sae.not.requested.cash.advance";
-    public static final String CONCUR_SAE_NOT_APPROVED_REQUESTED_CASH_ADVANCE_DATA_LINE = "validation.error.concur.sae.not.cash.admin.approved";
+    public static final String CONCUR_SAE_NOT_APPROVED_REQUESTED_CASH_ADVANCE_DATA_LINE = "validation.error.concur.sae.not.approved.or.applied";
```

Comment:
Should the name of this constant be updated to reflect the "approved-or-applied" change in the property name?